src/drive/web/modules/selection/duck.js

```diff
@@ -36,10 +33,9 @@ const selected = (state = [], action) => {
   }
   switch (action.type) {
     case SELECT_ITEM:
-      return [...state, action.id]
+      return uniqBy([...state, action.file], 'id')
```

Comment:
Can we add a todo to say that we should always index / normalize by id? In order to not having to deal with uniq / map filter?

Reply:
Again, it seems like a Map(id => object) would be better suited as the data structure for the selected slice.